tutorials/vue/vue-websocket.md

```diff
@@ -0,0 +1,47 @@
+[WebSockets](https://masteringjs.io/tutorials/node/websockets) are a great tool for when you want to show real time changes in data.
```

Comment:
For links to masteringjs.io, no need to put `https://masteringjs.io`, just `/tutorials/node/websockets` is the right way to go.